src/Service/OS2Forms/SubmissionNormalizers/AbstractSubmissionNormalizer.php

```diff
@@ -110,6 +110,8 @@ private function handleDocuments(string $sender, array $documentIds): array
                 // Create document
                 $document = $this->documentUploader->createDocumentFromPath($newFilePath, basename($documentUrl), 'OS2Forms');
 
+                unlink($newFilePath);
+
```

Comment:
Use `try … finally` here.

Reply:
Updated in b9c8983570afa1efd12f68ce397d899315fb2fe1.